index.html

```diff
@@ -304,19 +306,24 @@ <h4><a href="./news/2019-07-02-Guelph-accessibility-conference.html">Floe at the
             </div>
 
             <footer class="floe-footer">
+                    <div class="floe-footer-icons">
+                            <div class="social-media-logos-container">
+                                <a href="https://twitter.com/FluidProject" target="_blank"><img class="social-media-logo" alt="Twitter" src="images/icon-twitter.svg" /></a>
+                                <a href="https://www.facebook.com/idrctoronto/" target="_blank"><img class="social-media-logo" alt="Facebook" src="images/icon-facebook.svg" /></a>
+                                <a href="https://www.instagram.com/ocaduniversity/"  target="_blank"><img class="social-media-logo" alt="Instagram" src="images/icon-instagram.svg" /></a>
```

Comment:
I don't think these are all correct, and not even sure we have social media accounts for all of these for this particular project.

Reply:
@jobara these social media accounts are for the inclusive design institue.

Reply:
@aashimgarg, many of the projects have their own sites and own social media. Although this may vary from project to project. It is best to determine if they are needed for this site, and which ones would be most appropriate if they are.